examples/carousel/ui/carousel.slint

```diff
@@ -13,7 +13,7 @@ export component Carousel inherits FocusScope {
     in property <length> itemWidth;
     in property <int> count: 0;
     private property <length> center-x: (root.width - Theme.size-big) / 2;
-    private property <duration> duation: Theme.duration-regular;
+    private property <duration> duration: Theme.duration-regular;
```

Comment:
Well spotted! Thanks :) Could you please also fix the same typo in line 66, so that the file compiles?

Reply:
No problem :)